activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -169,6 +169,7 @@ def primary_key(name, type = :primary_key, **options)
         :date,
         :datetime,
         :decimal,
+        :numeric,
```

Comment:
@sgrif I've changed this to just add an alias for decimal but I think the error message about the precision error is now confusing. If someone try to create column in numeric type we will generate an error which says `Error adding decimal column: ...`. I think for someone who doesn't know numeric is just alias of decimal this message will be confusing.